tests/async_tests/test_interfaces.py

```diff
@@ -2,11 +2,14 @@
 import ssl
 
 import pytest
+import sniffio
 
 import httpcore
 from httpcore._types import URL
 from tests.conftest import Server, detect_backend
 
+pytestmark = pytest.mark.anyio
```

Comment:
I guess we'd rather keep the decorator style, with `@pytest.mark.anyio`? I know in the past we ended up eg having some case that are fine running on any one async environment (eg just asyncio), and in that case we could just switch to `@pytest.mark.asyncio`. (I also _personally_ prefer this "decorate every test case explicitly" for its, well, explicitness…)

Reply:
Switched back to the decorator style.